sdk/eventhub/event-hubs/package.json

```diff
@@ -78,20 +78,22 @@
     "typings/src"
   ],
   "scripts": {
-    "tslint": "tslint -p . -c tslint.json",
+    "build-browser": "tsc -p . && cross-env ONLY_BROWSER=true rollup -c 2>&1",
+    "build-node": "tsc -p . && cross-env ONLY_NODE=true rollup -c 2>&1",
+    "build-samples": "cd examples && tsc -p .",
+    "build-test": "tsc -p . && cross-env ONLY_NODE=true rollup -c rollup.test.config.js 2>&1",
+    "build": "tsc -p . && rollup -c 2>&1",
     "check-format": "prettier --list-different --config .prettierrc.json \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
+    "clean": "rimraf dist dist-esm test-dist typings *.tgz *.log",
+    "coverage": "npm run build-test && nyc --reporter=lcov mocha -t 120000 test-dist/index.js",
+    "extract-api": "tsc -p . && api-extractor run --local",
     "format": "prettier --write --config .prettierrc.json \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
-    "tsc": "tsc",
-    "prebuild": "rimraf dist && rimraf typings",
-    "build": "npm run tslint && tsc -p . && rollup -c",
-    "build-browser": "tsc -p . && cross-env ONLY_BROWSER=true rollup -c",
-    "build-node": "tsc -p . && cross-env ONLY_NODE=true rollup -c",
-    "build-test": "tsc -p . && cross-env ONLY_NODE=true rollup -c rollup.test.config.js",
-    "build-samples": "cd examples && tsc -p .",
+    "lint": "tslint -p . -c tslint.json",
+    "pack": "npm pack 2>&1",
+    "prebuild": "npm run clean",
```

Comment:
Good question. `prebuild` is not documented here: https://docs.npmjs.com/misc/scripts so I'm not sure if it will be run automatically at all. It might just be a confusing name. I can track down the originator of this line and ask them.

Reply:
@mikeharder I just tested locally and this doesn't get invoked when "build" is run. So I guess it's just a confusing name. It seems to have been added to this particular library by @amarzavery in b153ade, but I see that same script in numerous packages. I'm inclined to just leave it since it doesn't seem to affect the build (at least with Rush).